app/actions/WalletLoaderActions.js

```diff
@@ -101,6 +101,24 @@ export const createWalletRequest = (pubPass, privPass, seed, existing) =>
       .catch(error => dispatch({ error, type: CREATEWALLET_FAILED }));
   };
 
+export const CREATEWATCHONLYWALLET_ATTEMPT = "CREATEWATCHONLYWALLET_ATTEMPT";
+export const CREATEWATCHONLYWALLET_FAILED = "CREATEWATCHONLYWALLET_FAILED";
+export const CREATEWATCHONLYWALLET_SUCCESS = "CREATEWATCHONLYWALLET_SUCCESS";
+
+export const createWatchOnlyWalletRequest = (extendedPubKey, pubPass ="") => (dispatch, getState) => {
+  dispatch({ type: CREATEWATCHONLYWALLET_ATTEMPT });
+  return createWatchingOnlyWallet(getState().walletLoader.loader, extendedPubKey, pubPass)
```

Comment:
Change this to `wallet.createWatchingOnly` as it's how we're trying to standardize usage of the wallet functions.

Reply:
I thought we were using the same name from the WalletLoaderServiceService and the request name in there is `createWatchingOnlyWallet`. Is that wrong?

Reply:
Yeah, sorry should have been more accurate.

You can keep the `createWatchingOnlyWallet`, it's just that instead of importing and referencing it directly (`import { createWatchingOnlyWallet, ... } from "wallet" }`, use the wallet alias that imports everything (`import * as wallet from "wallet"`). 

We should eventually change all references to "dangling" functions to use the same style of module reference for consistency.